src/uniprotkb/config/apiUrls.ts

```diff
@@ -12,8 +12,8 @@ import { SortableColumn } from '../types/columnTypes';
 
 const devPrefix = 'https://wwwdev.ebi.ac.uk';
 const prodPrefix = 'https://www.ebi.ac.uk';
-const covidPrefix = 'https://www.ebi.ac.uk/uniprot/api/covid-19';
-// const covidPrefix = 'http://wp-np2-be:8090/uniprot/api/';
+// const covidPrefix = 'https://www.ebi.ac.uk/uniprot/api/covid-19';
+const covidPrefix = 'http://wp-p2m-be:8090/uniprot/api/covid-19';
```

Comment:
Should that end up in master?

Reply:
I need to temporarily make requests to this server while the fallback server is being used. Then when production is ready to be switched over to I will push to master with the normal www.ebi.ac.uk.

This isn't ideal but just doing this for now. Probably best to create a branch to test on fallback where `covidPrefix` is passed in for us at the build stage in the gitlab CI. I will make a jira for future updates.